.gitignore

```diff
@@ -11,3 +11,6 @@ debian/debhelper-build-stamp
 debian/files
 debian/nu.substvars
 debian/nu/
+
+# macOS junk
+.DS_Store
```

Comment:
@mhmdanas I highly recommend [setting up a global gitignore](https://gist.github.com/subfuzion/db7f57fff2fb6998a16c) for yourself, so you don't have to avoid doign this in all the projects you work on 🙂

Reply:
TIL there's a global gitignore! I'll add it to my gobal gitignore, but I still think it's a good idea to add .DS_Store to the project-level one because not everyone will have it in their global gitignore.